frontend/src/Components/Table/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+
+import { DataItem, Categories } from "../../dataUtilities";
+
+import styles from "./index.module.scss";
+
+interface Props {
+  data: Array<DataItem>;
+  categories: Array<Categories>;
+  identity: boolean;
+}
+
+const Table = (props: Props) => {
+  const maxRows = 15;
+  // regular table
+  const headers = props.categories.map((category) => (
+    <th className={styles.headerField}>{category.title}</th>
+  ));
+
+  let rows = props.data
+    .map((item: DataItem | any) => (
+      <tr className={styles.dataRows}>
+        {props.categories.map((category: Categories) => (
+          <td className={styles.dataField}>{item[category.field]}</td>
+        ))}
+      </tr>
+    ))
+    .slice(0, maxRows);
+
+  //identity table to accomodate odd data structure of identity product
+  const identityHeaders = props.categories.map((category) => (
+    <span className={styles.identityHeader}>{category.title}</span>
+  ));
+
+  const identityRows = props.data.map((item: DataItem | any) => {
+    return (
+      <div className={styles.identityDataRow}>
+        {props.categories.map((category: Categories) => (
+          <span className={styles.identityDataField}>
+            {item[category.field]}
+          </span>
+        ))}
+      </div>
+    );
+  });
```

Comment:
we dont need to construct these variables unless props.identity exists right? could we move them inside the if? or even make it its own component?

Reply:
i made a separate component